src/services/tasks.service.ts

```diff
@@ -8,4 +9,70 @@ export const getTaskByStatus = async (status: string): Promise<Task[]> => {
     return await Task.findAll({
         where: { status }
     });
+}
+
+export const getTaskByID = async (id: string): Promise<Task> => {
```

Comment:
```suggestion
export const getTaskByID = async (id: string): Promise<Task> => {

    const task = await Task.findOne({
        where: { id }
    });

    if (!task) {
        throw ('Could not find task ID in database');
    }

    return task
}
```
I am not a fan of throwing exception here when could not find the task by its id. The logic to handle that case should be in the upper level (in the function that calling this function `getTaskbyId`)